### PR TITLE
Feature/new pause panel

### PR DESCRIPTION
**Pull request by Gursaanj**
Changed the In-game Pause Panel to mirror the mock ups on Figma
Also replaced reference for button Images for the Options and Quit panel in the main menu
gif

### CODE REVIEW

**Comment by Gursaanj** [comment]
[Wrap up pause panel](https://trello.com/c/Zo7gNdf7/451-wrap-up-pause-panel)

File: Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanelBehaviour.cs [hunk 1]
```diff
@@ -39,14 +38,6 @@ private void ResetToDefault()
         m_SFXSlider.value = m_optionPanel.DefaultSoundEffectsVolume;
     }
 
-    private void OptionsPanelOnUpdate()
```
Comment (m4rtint) [inline Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanelBehaviour.cs:42]: Put this back.

Comment (Gursaanj) [reply Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanelBehaviour.cs:42]: Is this being called by Doozy? Cause There is no reference to this in the scripts

File: Assets/Doozy/MainMenu.asset [hunk 2]
```diff
@@ -477,81 +477,6 @@ MonoBehaviour:
...
-  m_minimumInputSocketsCount: 1
-  m_minimumOutputSocketsCount: 1
-  m_graphId: bdd95c85-1ef7-41c0-a017-1ece55e10d02
-  m_id: 9a2b4f24-fea1-4a89-941a-176161ee76be
-  m_name: SubGraph
-  m_notes: 
-  m_subGraph: {fileID: 0}
-  ErrorNoGraphReferenced: 1
-  ErrorReferencedGraphIsNotSubGraph: 1
 --- !u!114 &11400000
```
Comment (m4rtint) [inline Assets/Doozy/MainMenu.asset:480]: What did you change in the Main Menu Graph for Doozy?

File: Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanelBehaviour.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanelBehaviour.cs:42]: I guess this isn't in use anymore, whatever calls it was removed

GameEventMessage.SendEvent("BackButton");

this event is sent to a Doozy component or graph.  and that handles it.

File: Assets/Doozy/MainMenu.asset (hunk 2 above)
Comment (m4rtint) [reply Assets/Doozy/MainMenu.asset:480]: Whatever, doesn't seem like there are any difference when i checked

**Pull request merged**